cli-output/src/cli_output.rs

```diff
@@ -1457,26 +1457,69 @@ impl fmt::Display for CliSupply {
 
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct CliFees {
+pub struct CliFeesInner {
     pub slot: Slot,
     pub blockhash: String,
     pub lamports_per_signature: u64,
-    pub last_valid_slot: Slot,
+    pub last_valid_slot: Option<Slot>,
 }
 
-impl QuietDisplay for CliFees {}
-impl VerboseDisplay for CliFees {}
+impl QuietDisplay for CliFeesInner {}
+impl VerboseDisplay for CliFeesInner {}
 
-impl fmt::Display for CliFees {
+impl fmt::Display for CliFeesInner {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         writeln_name_value(f, "Blockhash:", &self.blockhash)?;
         writeln_name_value(
             f,
             "Lamports per signature:",
             &self.lamports_per_signature.to_string(),
         )?;
-        writeln_name_value(f, "Last valid slot:", &self.last_valid_slot.to_string())?;
-        Ok(())
+        let last_valid_slot = self
+            .last_valid_slot
+            .map(|s| s.to_string())
+            .unwrap_or_default();
+        writeln_name_value(f, "Last valid slot:", &last_valid_slot)
```

Comment:
nits: at this moment, we know `Last valid slot` doesn't work reliably. How about moving it under `--verbose`?

Reply:
Conversely, I'd like to see the base/current slot and base/current block height if these are easily available.

Okay, I'm starting to asking more and more from your small pr. Don't take this serious.. xD

Reply:
> nits: at this moment, we know `Last valid slot` doesn't work reliably. How about moving it under `--verbose`?

I am not in favor of this, until we have a broader fix (eg last-valid-block-height).